package.json

```diff
@@ -27,12 +43,13 @@
   },
   "files": [
     "lib",
+    "dist",
     "cuid.d.ts",
-    "index.js"
+    "index.esm.js"
   ],
   "license": "MIT",
   "devDependencies": {
-    "browserify": "17.0.0",
+    "@rollup/plugin-alias": "^5.1.1",
```

Comment:
is this dependency being used?

Reply:
this handles the `#fingerprint` import flipping between node and browser versions